.travis.yml

```diff
@@ -4,6 +4,10 @@ node_js: "5.5.0"
 
 sudo: false
 
+env:
+  GITHUB_REPO: github.com/18F/fec-style
```

Comment:
Sure, would you recommend changing the script or travis.yml? I think this looks a bit funny:

    GITHUB_REPO: github.com/$TRAVIS_REPO_SLUG